bin/sl-meshctl.txt

```diff
@@ -84,6 +84,10 @@ services using `ls` or `status`.
         a grace period for any existing connections to finish. "Hard" stops
         kill the supervisor and its workers with `SIGTERM`.
 
+  tracing-start SVC       Restart all workers with tracing on.
```

Comment:
Eventually, this should be `SVC | SVC.EXEC`, but this is fine for strong-pm, where its the same.